madgraph.sh

```diff
@@ -10,7 +10,41 @@ build_requires:
 ---
 #!/bin/bash -e
 
-rsync -a --exclude='**/.git' --delete --delete-excluded $SOURCEDIR/ $INSTALLROOT/
+rsync -a --exclude='**/.git' --delete --delete-excluded $SOURCEDIR/ $BUILDDIR/
+
+# install internal packages 
+cd $BUILDDIR
+cat << EOF >> $PWD/install.dat
+install oneloop
+install ninja
```

Comment:
can't you reuse our own ninja? Or is this a separate ninja?

Reply:
This is a different "ninja", it is not the build tool, it is a tool used in one-loop integration in NLO processes: https://ninja.hepforge.org. Without this OneLoop as loop tool (and consequently madgraph) does not work.